app/src/main/java/com/naman14/timberx/TimberMusicService.kt

```diff
@@ -285,7 +286,7 @@ class TimberMusicService : MediaBrowserServiceCompat(), MediaPlayer.OnPreparedLi
             override fun onStop() {
                 player?.stop()
                 setPlaybackState(mStateBuilder.setState(STATE_NONE, 0, 1F).build())
-                NotificationUtils.updateNotification(this@TimberMusicService, mMediaSession)
+                notifications.updateNotification(mMediaSession)
```

Comment:
Notice that you no longer need to pass a `Context` because `Notifications` is able to get the application context during injection of itself